src/view/reminder/commons/reminder-time-options.ts

```diff
@@ -3,79 +3,196 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
 import { TFunction } from 'i18next';
+import { filter } from 'lodash';
+import moment from 'moment';
+import { useMemo } from 'react';
+import { EventType } from '../../../types/appointment-reminder';
+
+type ReturnType = Array<{
+	id: string;
+	label?: string;
+	click?: () => void;
+	value: number;
+	type?: string;
+}>;
+export const useGetReminderItems = (
+	t: TFunction,
```

Comment:
Can "t" be instantiated inside the hook?

Reply:
Yes it can,I'll update it